src/test/java/com/bandwidth/sqs/consumer/ConsumerManagerTest.java

```diff
@@ -24,16 +32,16 @@
     private SqsConsumerManager consumerManager;
 
     public ConsumerManagerTest() {
-        consumerManager = new SqsConsumerManager(MAX_LOAD_BALANCED_REQUESTS, threadPoolMock);
+        consumerManager = new SqsConsumerManager(MAX_LOAD_BALANCED_REQUESTS, threadPoolMock, NUM_PERMITS);
         consumerManager.addConsumer(consumerMock);
         consumerManager.addConsumer(consumerMock2);
     }
 
     @Test
     public void queueReadyConsumerTest() {
-        Runnable task = mock(Runnable.class);
-        consumerManager.queueTask(task);
-        verify(threadPoolMock).execute(task);
+        Function<SqsMessage, Completable> task = mock(Function.class);
```

Comment:
I could I guess, it does the same thing.
`mock(Function.class)` is shorter than
`(arg) -> Completable.complete()`